inspec.yml

```diff
@@ -0,0 +1,10 @@
+name: postgres-hardening
+title: Hardening Framework Postgres Hardening Test Suite
+maintainer: Hardening Framework Team
+copyright: Hardening Framework Team
+copyright_email: hello@dev-sec.io
+license: Apache 2 license
+summary: Test-suite for best-practice postgres hardening
+version: 0.1.0
```

Comment:
I think we should bump it to version 2.0.0 since 1.x is the Serverspec-based implementation